### PR TITLE
🎉 Migrate Confluence Source Connector to Low Code

### LINKED ISSUE #28989: confluence
Please use the [connector builder](https://docs.airbyte.com/connector-development/config-based/low-code-cdk-overview/?_gl=1*13hxonw*_gcl_au*MTI0NTMzNjM3OS4xNjkwODM4MzE0*_ga*MTg2NTc3OTc0Ni4xNjUxNTk0NTQw*_ga_HDBMVFQGBH*MTY5MTAwNDEzMy4yNzIuMC4xNjkxMDA0MTMzLjAuMC4w&_ga=2.197654767.1978878.1690818922-1865779746.1651594540) for this.

### PR DESCRIPTION
**Pull request by topefolorunso**
## What
Migrating Source Confluence to Low-Code CDK resolves #28989

## How
Created the connector using the connector buider UI, then imported the manifest.yaml to the low code cdk.

## Recommended reading order
1. `manifest.yaml`

## 🚨 User Impact 🚨
*Are there any breaking changes? What is the end result perceived by the user?*

*For connector PRs, use this section to explain which type of semantic versioning bump occurs as a result of the changes. Refer to our [Semantic Versioning for Connectors](https://docs.airbyte.com/contributing-to-airbyte/#semantic-versioning-for-connectors) guidelines for more information. **Breaking changes to connectors must be documented by an Airbyte engineer (PR author, or reviewer for community PRs) by using the [Breaking Change Release Playbook](https://docs.google.com/document/d/1VYQggHbL_PN0dDDu7rCyzBLGRtX-R3cpwXaY8QxEgzw/edit).***

*If there are breaking changes, please merge this PR with the 🚨🚨 emoji so changelog authors can further highlight this if needed.*

## Pre-merge Actions
*Expand the relevant checklist and delete the others.*

<details><summary><strong>Updating a connector</strong></summary>

### Community member or Airbyter

- Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- Unit & integration tests added

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- Create a non-forked branch based on this PR and test the below items on it
- Build is successful
- If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).

</details>

### CODE REVIEW

File: airbyte-integrations/connectors/source-confluence/Dockerfile [hunk 1]
```diff
@@ -34,5 +34,5 @@ COPY source_confluence ./source_confluence
 ENV AIRBYTE_ENTRYPOINT "python /airbyte/integration_code/main.py"
 ENTRYPOINT ["python", "/airbyte/integration_code/main.py"]
 
-LABEL io.airbyte.version=0.1.3
+LABEL io.airbyte.version=0.1.0
```
Comment (sajarin) [inline airbyte-integrations/connectors/source-confluence/Dockerfile:37]: Can you update the version of the connector appropriately here? It looks like you're downgrading here.

File: airbyte-integrations/connectors/source-confluence/metadata.yaml [hunk 2]
```diff
@@ -2,24 +2,26 @@ data:
...
+  registries:
+    oss:
+      enabled: true
+    cloud:
+      enabled: true
   connectorSubtype: api
   connectorType: source
   definitionId: cf40a7f8-71f8-45ce-a7fa-fca053e4028c
-  dockerImageTag: 0.1.3
+  dockerImageTag: 0.1.0
```
Comment (sajarin) [inline airbyte-integrations/connectors/source-confluence/metadata.yaml:13]: Same comment as above regarding the version

File: airbyte-integrations/connectors/source-confluence/metadata.yaml [hunk 3]
```diff
@@ -2,24 +2,26 @@ data:
...
   license: MIT
   name: Confluence
-  registries:
-    cloud:
-      enabled: true
-    oss:
-      enabled: true
+  releaseDate: TODO
   releaseStage: beta
+  supportLevel: certified
```
Comment (sajarin) [inline airbyte-integrations/connectors/source-confluence/metadata.yaml:21]: supportLevel should be "community"

File: airbyte-integrations/connectors/source-confluence/metadata.yaml [hunk 4]
```diff
@@ -2,24 +2,26 @@ data:
...
   githubIssueLabel: source-confluence
   icon: confluence.svg
   license: MIT
   name: Confluence
-  registries:
-    cloud:
-      enabled: true
-    oss:
-      enabled: true
+  releaseDate: TODO
```
Comment (sajarin) [inline airbyte-integrations/connectors/source-confluence/metadata.yaml:19]: Can you update the release date with 2021-11-05? (This is when the connector was first published based on our docs)

File: airbyte-integrations/connectors/source-confluence/source_confluence/schemas/TODO.md
Comment (sajarin) [inline airbyte-integrations/connectors/source-confluence/source_confluence/schemas/TODO.md:1]: Can you remove this file?

**Review by sajarin — CHANGES_REQUESTED** [review]
Thanks for the PR, left some comments:

**Review by sajarin — CHANGES_REQUESTED** [review]
Can you update the schema files with "additionalProperties" set to true?

**Comment by topefolorunso** [comment]
> Can you update the schema files with "additionalProperties" set to true?

Fixed @sajarin

**Comment by topefolorunso** [comment]
> @topefolorunso there are some validation errors in the manifest.yaml. Please correct them and then attach screenshots of passing acceptance tests

Validation error fixed @sajarin 

image

**Comment by sajarin** [comment]
@topefolorunso some more errors after running the tests: 

image

image

**Comment by topefolorunso** [comment]
@sajarin Errors should be fixed now.

- The audit stream requires a premium account which I don't have. So I was unable to test the stream. Kindly test again and let me know if anything else is broken.
- The pk used previously was the "author" property which is not unique. Apparently, there is no unique id for each audit log, so I opted for the "creationDate" property which is expected to be unique. (I hope that is okay)
- Finally, for some reason, I have been unable to run integration tests for all my connectors, probably due to some recent changes in the codebase. Please advise.

**Comment by sajarin** [comment]
Still running into some issue with the primary keys but running the tests in a separate PR as per our usual review process.

**Review by sajarin — APPROVED** [review]

**Comment by sajarin** [comment]
Thanks for the contribution @topefolorunso!

**Pull request merged**